src/Museum/Museum.php

```diff
@@ -2,7 +2,38 @@
 
 namespace DinoApp\Museum;
 
+use DinoApp\Dinosaur\Dinosaur;
+
 class Museum
 {
+    /**
+     * Takes in an array of dinosaur objects and puts them into the format to be displayed on the homepage.
+     *
+     * @param array $dinos an array of Dinosaur objects.
+     * @return string The html to display the dinosaurs on the homepage.
+     */
+    public static function displayAllDinos(array $dinos) :string
+    {
+        $output='';
+        foreach ($dinos as $dino){
+            if($dino instanceof Dinosaur){
+                $output .= '<style> .dino'. $dino->getId().'{background-image: url(' . $dino->getImageUrl() .');} </style>';
```

Comment:
I'd actually do this as a style attribute on dino-img-container div rather than as a style tag. Just because it means fewer elements in the DOM